overrides/windows-override.json

```diff
@@ -77,12 +77,11 @@
             }
         },
         "autoconsent": {
-            "state": "disabled",
+            "state": "enabled",
             "minSupportedVersion": "0.50.0",
-            "settings": {
```

Comment:
You might want to keep the `settings` node, as the [AutoConsent.Service](https://github.com/duckduckgo/windows-browser/blob/657473e4a55ab81d019a7746d117ea1e1f01d53b/WindowsBrowser/Application/Autoconsent/Service.cs#L103) relies on its existence.

The updated config still seems have a `settings` node, though, with:

```
disabledCMPs ...
```

Reply:
As discussed in MM, I made manual edits on my local config. I didn't consider the base config, which I could have. Thanks for resolving this.

Reply:
> as the settings section, and this file is supposed to be applied on top of that

These won't merge. Is this your intention?

Reply:
Ignore the last comment, I was confused by the thread :)